core/build.gradle

```diff
@@ -47,4 +47,25 @@ dependencies {
     androidTestImplementation dependency.androidx.test.core
     androidTestImplementation dependency.androidx.test.runner
     androidTestImplementation dependency.androidx.test.junit
+    androidTestImplementation dependency.androidx.test.espresso
+    androidTestImplementation dependency.androidx.nav.navtesting
+
+    debugImplementation dependency.androidx.nav.fragmenttesting
```

Comment:
Shouldn't this say `androidTestImplementation`? `debugImplementation` is an `implementation` for the `debug` build. If you need it in the main artifact (not the test artifact), I'd sort this up int othe `implementation` section, as opposed to the instrument test dependencies section, where it is now.